### PR TITLE
add option to opt into 'scale' backend

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team 

image

For dev's locally:
image

### CODE REVIEW

**Comment by snopoke** [comment]
https://confluence.dimagi.com/display/ccinternal/Scale+Backend+Beta:+Information+and+Caveats

File: corehq/apps/registration/forms.py [hunk 1]
```diff
@@ -35,23 +35,42 @@ class DomainRegistrationForm(forms.Form):
...
 
+    use_new_backend = forms.MultipleChoiceField(
+        required=False,
+        choices=(
+            ('yes', _("Opt into beta testing of the new 'scale' backend (Dimagi only)")),
+        ),
+        widget=forms.CheckboxSelectMultiple,
+        help_text=mark_safe(
+            "<p class=\"text-danger\">Read this first: "
+            "<a href=\"https://confluence.dimagi.com/display/ccinternal/Scale+Backend+Beta:+Information+and+Caveats\">"
```
Comment (dimagimon) [inline corehq/apps/registration/forms.py]: E501 line too long (119 > 115 characters)

File: corehq/toggles.py [hunk 2]
```diff
@@ -835,3 +835,9 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_PRODUCT_PATH,
     [NAMESPACE_DOMAIN]
 )
+
+
+def _enable_toggles_for_scale_beta(domain):
```
Comment (benrudolph) [inline corehq/toggles.py]: nit: maybe not prefixed with `_` since it's being imported elsewhere

**Comment by benrudolph** [comment]
it's happening!

**Comment by czue** [comment]
woop woop! :elephant:

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Comment by NoahCarnahan** [comment]
http://i.imgur.com/7drHiqr.gif

**Comment by dannyroberts** [comment]
awesome!

File: corehq/apps/registration/forms.py [hunk 3]
```diff
@@ -36,23 +36,43 @@ class DomainRegistrationForm(forms.Form):
...
 
+    use_new_backend = forms.MultipleChoiceField(
+        required=False,
+        choices=(
+            ('yes', _("Opt into beta testing of the new 'scale' backend (Dimagi only)")),
+        ),
+        widget=forms.CheckboxSelectMultiple,
+        help_text=mark_safe(
+            "<p class=\"text-danger\">Read this first: "
+            "<a href=\"https://confluence.dimagi.com/display/ccinternal/Scale+Backend+Beta:+Information+and+Caveats\"  target=\"_blank\">"
```
Comment (dimagimon) [inline corehq/apps/registration/forms.py]: E501 line too long (138 > 115 characters)

**Comment by snopoke** [comment]
:heavy_check_mark: once build passes (though not sure this has any test coverage and I did test it locally)

**Comment by snopoke** [comment]
🙌🎉

**Pull request merged**